tests/target-browser/target_browser_suite_test.go

```diff
@@ -175,17 +176,21 @@ func createTarget(enableBrowsing bool) {
 }
 
 func deleteTarget(enableBrowsing bool) {
-	if !enableBrowsing {
+	if enableBrowsing {
 		Expect(updateYAMLs(map[string]string{"enableBrowsing: false": "enableBrowsing: true"}, targetYamlPath)).To(BeNil())
 	}
+	target := getTarget(ctx, installNs, k8sClient)
+	pvcName, isPresent, _ := unstructured.NestedString(target.Object, "status", "nfsPersistentVolumeClaim", "name")
```

Comment:
instead of this, GET target's PVC by label `app.kubernetes.io/instance: <target-name>`